temporal-normalizer/src/test/java/edu/illinois/cs/cogcomp/temporal/normalizer/tests/TestTemporalChunker.java

```diff
@@ -0,0 +1,354 @@
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+    }
+
+    private static final String textFile = "text/AP_20130322";
+
+    @Test
+    public void testPosPipeline() {
+        String input = null;
+        try {
+            input = LineIO.slurp(textFile);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+            fail(e.getMessage());
+        }
+
+        assertFalse(input.length() == 0);
+
+        AnnotatorService as = null;
+        try {
+            as = PipelineFactory.buildPipeline(ViewNames.POS);
+        } catch (IOException | AnnotatorException e) {
+            e.printStackTrace();
+            fail(e.getMessage());
+        }
+
+        String norm = input.replaceAll("\\n", " ");
+        norm = StringCleanup.normalizeToUtf8(norm);
+
+
+        try {
+            as.createAnnotatedTextAnnotation("test", "test", norm);
+        } catch (AnnotatorException e) {
+            e.printStackTrace();
+            fail(e.getMessage());
+        }
+    }
+
+    public void testNormalizationWithTrueExtraction() throws Exception {
+        Properties props = new Properties();
+        props.setProperty( PipelineConfigurator.USE_POS.key, Configurator.TRUE );
+        // TODO: Configurtaor.TRUE/FALSE is reversed, in the future when this is fixed,
+        // TODO: change it back
+        props.setProperty( PipelineConfigurator.USE_SENTENCE_PIPELINE.key, Configurator.TRUE );
+        AnnotatorService pipeline = PipelineFactory.buildPipeline(new ResourceManager(props));
+
+        System.out.println("Working Directory = " +
+                System.getProperty("user.dir"));
+        ResourceManager nerRm = new TemporalChunkerConfigurator().getDefaultConfig();
+        IOUtilities.existsInClasspath(TemporalChunkerAnnotator.class, nerRm.getString("modelDirPath"));
+
+        java.util.logging.Logger.getLogger("HeidelTimeStandalone").setLevel(Level.OFF);
+
+        long preprocessTime = System.currentTimeMillis();
+        List <TextAnnotation> taList = new ArrayList<>();
+        for (int j = 0; j < te3inputText.size(); j ++) {
+            String filename = "./text/"+docIDs.get(j);
+            PrintStream ps = new PrintStream(filename);
+            ps.print(testText.get(j));
+            ps.close();
+
+            FileInputStream fis = new FileInputStream(filename);
+            int content;
+            String text = "";
+            while ((content = fis.read()) != -1) {
+                // convert to char and display it
+                text += (char) content;
+            }
+
+            text = text.replaceAll("\\n", " ");
+            TextAnnotation ta = pipeline.createAnnotatedTextAnnotation( "corpus", "id",  text);
+            taList.add(ta);
+        }
+
+        long startTime = System.currentTimeMillis();
+
+        int numTimex = 0;
+
+        String outputFolder = "AQ_goldext_illininorm";
```

Comment:
document this, have it set as an input parameter (either in function call, main argument, or configurator field)